src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml

```diff
@@ -448,7 +276,6 @@
 
                                     <!--Found classes-->
                                     <ListView Name="SubCategoryListView"
-                                              ItemTemplateSelector="{StaticResource ClassObjectTemplateSelector}"
```

Comment:
SubCategoryListView is found classes during search. Since classes are hidden, we can clean code a little bit and remove selector and datatemplates.